generators/app/prompts.js

```diff
@@ -117,157 +104,29 @@ function askForOperations(generator) {
     const prompts = [
         {
             when: !generator.options['skip-prompts'],
-            type: 'list',
-            name: 'generationType',
-            message: 'Which type of generation do you want?',
-            choices: generationTypeChoices(),
-            default: [constants.BIGBANG_MODE]
+            type: 'confirm',
+            name: 'cleanup',
+            message: 'Do you want to clean up your current Kafka configuration?',
+            default: false,
+            validate: input => (_.lowerCase(input) !== 'y' && _.lowerCase(input) !== 'n' ? 'Please enter Y or N' : true)
```

Comment:
Finally after testing it `validate` does not work because there is `default: false` so you can remove `validate`.

Reply:
i removed it